sniffr/models.py

```diff
@@ -93,6 +93,41 @@ def __init__(
     def __repr__(self):
         return f"<Dog: {self.dog_name} | Age: {self.age}>"
 
+class Activity(db.Model):
+    __tablename__ = "activity"
+
+    activity_id = db.Column(db.Integer, primary_key=True)
+    activity_description = db.Column(db.Text(), nullable=False)
+
+    def __init__(self, activity_id, activity_description):
+        self.activity_id = activity_id
+        self.activity_description = activity_description
+        
+    def __repr__(self):
+        return f"<Description: {self.activity_description}>"
+
+class DogActivies(db.Model):
+    __tablename__ = "dog_activities"
+
+    dog_activities_id = db.Column(db.Integer, primary_key=True)
```

Comment:
```suggestion
    dog_activity_id = db.Column(db.Integer, primary_key=True)
```
Each record is a singular dog_activity event

Reply:
See the comment on models.py.

Our model uses a plural `dog activities` table to reference a singular `activity` lookup table. I'm not hard set on this practice, but we shouldn't deviate from it in the code.